sale_order_packaging_import/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module extends the `sale_order_import` module to include on the sale orde line
```

Comment:
worth to mention that you assume that the GTIN provided is the one of the packaging instead of the product one

Reply:
I updated the description, I did not mention gtin but gave a better explanation of how it works